chart/openfaas/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Enable Kubernetes as a backend for OpenFaaS (Functions as a Service)
 name: openfaas
-version: 1.1.0
+version: 1.1.1
```

Comment:
I pushed this up with a namespace change to openfaas/ on the Hub, so it needs a bump to 1.1.2

Reply:
This is the chart version, how is related to `openfaas/` Docker Hub?

Reply:
Ah got it